app/src/full/res/raw/wear_keep.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:keep="@array/android_wear_capabilities"/>
```

Comment:
Any reason to put it in `full` flavor and not `main`?

Reply:
I thought main didn't include the capabilities, only full. I'll confirm and move.

In truth we could move this to common and have a single keep file.

Reply:
Indeed you are right that wear only works on `full` 👍🏻